enhancements/microshift/microshift-default-csi-plugin.md

```diff
@@ -143,10 +143,7 @@ The workflow is automated and documented
 
 > After the device owner has logged into the device ... 
 
-1. The device owner will create a file called `lvmd.yaml`, per documentation according to its 
-[documentation](https://github.com/red-hat-storage/topolvm/blob/main/docs/lvmd.md).
-   1. The device owner will provide the absolute path to this file in the MicroShift config field `LvmdConfigFilePath`. 
-
+1. The device owner will create a file called `lvmd.yaml` _in the same directory as the https://github.com/openshift/microshift/blob/main/docs/howto_config.md_, specified per the lvmd [documentation](https://github.com/red-hat-storage/topolvm/blob/main/docs/lvmd.md).
```

Comment:
It looks like the URL is in there twice, is that intentional? Or should that read "in the same directory as the config file"?

Reply:
Ah, will fix